CHANGELOG-3.3.md

```diff
@@ -10,6 +10,10 @@ The minimum recommended etcd versions to run in **production** are 3.2.28+, 3.3.
 
 ## v3.3.18 (2019-11-26)
 
+### etcd server
+
+- [Fix corruption bug in defrag](https://github.com/etcd-io/etcd/pull/11613).
```

Comment:
Oh, wait, are we backporting this to 3.3.19?

Reply:
oops, fixing

Reply:
Done